extensions/podman/packages/extension/src/remote/podman-remote-connections.ts

```
@@ -139,58 +137,45 @@ export class PodmanRemoteConnections {
-        localPath = join(tmpdir(), `podman-remote-${connection.Name}.sock`);
+      try {
+        const uri = new URL(connection.URI);
+        const host = uri.hostname;
+        const port = parseInt(uri.port);
+        const username = uri.username;
+        const privateKeyFile = connection.Identity;
+
+        const privateKey = readFileSync(privateKeyFile, 'utf8');
+        const remotePath = uri.pathname;
+
+        let localPath;
```

Biome (2.5.0)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)